ffv1.md

```diff
@@ -476,7 +474,7 @@ RFC:```
 RFC:j_{0} = 2
 RFC:```
 
-#### Range non binary values
+#### Range Non Binary Values
```

Comment:
I am not an English speaker: `Non Binary` or `Non-Binary`?

Reply:
IIRC hyphens presents in section headers break the inter-sectional linking as the document is processed through mmark and xml2rfc.

Reply:
Thank you for the explanation, @dericed!